doc/man/8/mount.ceph.rst

```diff
@@ -23,170 +26,193 @@ client component does most of the real work. In fact, it is possible
 to mount a non-authenticated Ceph file system without mount.ceph by
 specifying monitor address(es) by IP::
 
-        mount -t ceph 1.2.3.4:/ mountpoint
+        mount -t ceph 1.2.3.4:/ /mnt/mycephfs
 
-Each monitor address monaddr takes the form host[:port]. If the port
-is not specified, the Ceph default of 6789 is assumed.
+The first argument is the device part of the mount command. It includes host's
+socket and path within CephFS that will be mounted at the mount point. The
+socket, obviously, takes the form ip_address[:port]. If the port is not
+specified, the Ceph default of 6789 is assumed.
 
-Multiple monitor addresses can be separated by commas. Only one
-responsible monitor is needed to successfully mount; the client will
-learn about all monitors from any responsive monitor. However, it is a
-good idea to specify more than one in case one happens to be down at
-the time of mount.
+Multiple monitor addresses can be passed by separating them by commas. Only
+one monitor is needed to successfully mount; the client will learn about all
+monitors from any responsive monitor. However, it is a good idea to specify
+more than one in case the one happens to be down at the time of mount.
 
 If the host portion of the device is left blank, then **mount.ceph** will
 attempt to determine monitor addresses using local configuration files
 and/or DNS SRV records.
 
-A subdirectory subdir may be specified if a subset of the file system
-is to be mounted.
+A subset of the file system can be mounted by specifying the (absolute) path
+to the sub-directory right after ":" after the sockets.
 
-Mount helper application conventions dictate that the first two
-options are device to be mounted and destination path. Options must be
+Mount helper application conventions dictate that the first two options are
+device to be mounted and the mountpoint for that device. Options must be
 passed only after these fixed arguments.
 
 
 Options
 =======
 
-:command:`wsize`
-  int (bytes), max write size. Default: 16777216 (16*1024*1024) (writeback uses smaller of wsize
-  and stripe unit)
+:command:`cap_release_safety`
+    int, Default: calculated
```

Comment:
I think it's fine to avoid going in to detail in the manpage, but let's reference the doc where this option is described. We could do that in bulk by describing grouping those options as "advanced options" or something like that at the bottom.

Reply:
"Advanced options" is a great idea.

Reply:
Agreed. We should document everything in the manpage.